environment.js

```diff
@@ -2,25 +2,37 @@ var path = require('path');
 var fs = require('fs');
 var mkdirp = require('mkdirp');
 var glob = require('glob');
+var q = require('q');
 
-export.ensureDirectory = function(directoryName) {
+exports.ensureDirectory = function(directoryName) {
   if (!fs.existsSync(directoryName)) {
     mkdirp.sync(directoryName);
   }
 };
 
-export.loadSpecifications = function(mocha, directory) {
-   function addSpecs(error, files) {
+
+exports.loadSpecifications= function(directory, mocha) {
+  var deferred = q.defer();
```

Comment:
Yeah, at one point I was passing around the file list as well but I think its better to return the mocha instance rather than the spec file paths.